client/index.html

```diff
@@ -7,11 +7,20 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>State Channel Demo</title>
     <meta property="og:type" content="website" />
-    <meta property="og:title" content="State Channels Demo" />
+    <meta property="og:title" content="State Channel Demo" />
     <meta
       property="og:description"
       content="The game of rock-paper-scissors implemented on the æternity blockchain using state channels"
     />
+    <meta property="og:image" content="/thumbnail.png" />
+    <meta name="twitter:card" content="summary_large_image" />
+    <meta name="twitter:site" content="@aeternity" />
+    <meta name="twitter:title" content="State Channel Demo" />
+    <meta
+      name="twitter:description"
+      content="The game of rock-paper-scissors implemented on the æternity blockchain using state channels"
+    />
+    <meta name="twitter:image" content="/thumbnail.png" />
```

Comment:
```suggestion
    <meta name="twitter:image" content="https://statechannel.aepps.com/rock-paper-scissors/thumbnail.png" />
```

It seems like twitter does not support relative links, see
- https://stackoverflow.com/a/45941439
- https://twittercommunity.com/t/card-error-unable-to-render-or-no-image-read-this-first/62736

![image](https://user-images.githubusercontent.com/45261205/205497245-52ff1095-dd60-4ffc-805b-c246fd7968dc.png)


Since there is no other way to fetch the image with any other way, i propose to use the suggested, full URL

Reply:
The proposed URL wouldn't work. But `https://statechannel.aepps.com/thumbnail.png` would. This would need to be updated once the demo actually moves to `https://statechannel.aepps.com/rock-paper-scissors/`.

Reply:
not sure if the demo moves to `/rock-paper-scissors`. I mean it's currently accessible through both URLs. so probably it doesn't matter.

proposed URL should work I guess 🤔

Reply:
It is only deployed on `https://statechannel.aepps.com` so the image would only exist here `https://statechannel.aepps.com/thumbnail.png`. 
I changed the URL but it will only work once it's deployed on production